tests/ui/impl-trait/in-trait/async-and-ret-ref.stderr

```diff
@@ -7,7 +7,10 @@ LL |     async fn foo() -> &'static impl T;
    |     the associated type `<Self as MyTrait>::{opaque#0}` must be valid for the static lifetime...
    |     ...so that the reference type `&'static impl T` does not outlive the data it points at
    |
-   = help: consider adding an explicit lifetime bound `<Self as MyTrait>::{opaque#0}: 'static`...
+help: consider adding an explicit lifetime bound
+   |
+LL |     async fn foo() -> &'static impl T + 'static;
```

Comment:
You mean because of the semantics or because of the lack of parentheses? I [did check that applying the suggestion would cause compilable code](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=bed72fa8ec763e293f38238c4d4ccb98). Whether the semantics of RPITIT should already handle this is a different matter.

Reply:
Wait, I didn't notice that this was an RPIT itself, nvm.